### DISCUSSION TITLE
data-filter Attribute for data-type="ajax"

### DISCUSSION THREAD

**Discussion by mmoollllee** [fancyapps/ui#155] [Q&A] [answered]
With fancybox 3 there was a `data-filter="#content"`-attribute to filter the shown HTML when used with `data-type="ajax"`.
Is something similar planned for fancybox 4 or already integrated?

**Comment by fancyapps** [comment, marked as answer]
Hi,

Sorry about the late reply, it was added in the previous release.

**Reply by mmoollllee** [reply]
Thaaaannnkkk youuu <3
